archive/tar_unix.go

```diff
@@ -23,6 +23,7 @@ func chmodTarEntry(perm os.FileMode) os.FileMode {
 	return perm
 }
 
+// nolint: unconvert
```

Comment:
What was the warning here?

Reply:
`s.Rdev` is already `uint64` on 64bit platforms, but the conversion is necessary for 32bit.

Following up on the above feedback, I think I'm going to try and find a way to both make it more obvious, and remove the need for unconvert.

Reply:
Correction, it's not 32bit, it's darwin-amd64 which uses int32.